fendermint/testing/graph-test/subgraph/graph-node/docker-compose.yaml

```diff
@@ -0,0 +1,46 @@
+version: '3'
+services:
+  graph-node:
+    image: graphprotocol/graph-node:v0.27.0
+    ports:
+      - '8000:8000'
+      - '8001:8001'
+      - '8020:8020'
+      - '8030:8030'
+      - '8040:8040'
+    depends_on:
+      - ipfs
+      - postgres
+    extra_hosts:
+      - host.docker.internal:host-gateway
+    environment:
+      postgres_host: postgres
+      postgres_user: 'graph-node'
+      postgres_pass: 'let-me-in'
+      postgres_db: 'graph-node'
+      ipfs: 'ipfs:5001'
+      ethereum: 'testnet:http://host.docker.internal:8545'
+      GRAPH_LOG: info
+      GRAPH_ETHEREUM_GENESIS_BLOCK_NUMBER: 1
+  ipfs:
+    image: ipfs/go-ipfs:v0.10.0
+    ports:
+      - '5001:5001'
+    volumes:
+      - ./data/ipfs:/data/ipfs
+  postgres:
+    image: postgres
+    ports:
+      - '5432:5432'
+    command:
+      [
+          "postgres",
+          "-cshared_preload_libraries=pg_stat_statements"
+      ]
+    environment:
+      POSTGRES_USER: 'graph-node'
+      POSTGRES_PASSWORD: 'let-me-in'
+      POSTGRES_DB: 'graph-node'
+      PGDATA: "/data/postgres"
+    volumes:
+      - ./data/postgres:/var/lib/postgresql/data
```

Comment:
We could also just not mount this for example. We don't need it to persist between restarts. Same for the IPFS container. Not sure why config could not be removed, though.

Reply:
Great suggestion. I removed the mount.